scripts/createPage/createPage.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable */
+const path = require('path');
+const fs = require('fs');
+const { buildJS } = require('./builders');
+const {
+  containsWhiteSpace,
+  containsDashCharacter,
+  capitalizeFirstLetter,
+} = require('../../common/utils/node-utils');
+
+const pagePath = 'pages/';
+const fileNamePrefix = '.js';
+
+const findRoot = () => {
+  let thisPath = path.resolve(__dirname);
+  while (!fs.existsSync(path.join(thisPath, 'package.json'))) {
+    thisPath = path.join(thisPath, '..');
+  }
+
+  return thisPath;
+};
+
+const doesPageExist = (pageName, root) => {
+  const newPath = path.join(root, pagePath, pageName);
+  const filePrefix = '.js';
+
+  if (fs.existsSync(newPath + filePrefix)) {
+    console.log(`Page "${pageName}" Already Exists`);
+    return true;
+  }
+  return false;
+};
+
+const createPageTitle = pageName => {
+  let capitalizePageName = pageName.split('_').map((word, index) => capitalizeFirstLetter(word));
+
+  if (capitalizePageName[0] === '') {
+    capitalizePageName = capitalizePageName.slice(1);
+  }
+
+  return capitalizePageName.join(' ');
+};
+
+const createPage = (root, pageName) => {
+  // test to make sure file meets requirements
+  if (containsDashCharacter(pageName) || containsWhiteSpace(pageName)) {
```

Comment:
so going off what I said earlier on the new utility functions, here you can just do:
```javascript
const isNotValidPageName = pageName.includes('-') || pageName.includes(' ');

if (isNotValidPageName) {
  console.log(`Page: ${pageName} wasn't created because of a dash or white space in name.`);
  return false;
}

//etc
```

Reply:
Good idea. Will make that change now.